Prometheus/Readme.md

```diff
@@ -1,5 +1,8 @@
-#Steps and commands
-
+# Prometheus
+You can learn Prometheus from the following blogs and youtube videos
+- [Blog](https://blogs.kaiwalyakoparkar.com/introduction-to-prometheus)
+ [![YouTube Video](https://img.youtube.com/vi/EcixeZzyaVc/hqdefault.jpg)](https://youtu.be/EcixeZzyaVc)
+# Steps and commands
```

Comment:
Please add one empty line before and after the heading please (for all the files)